Core/NoOpRunner.Core/Shapes/MovingShape.cs

```diff
@@ -41,7 +41,17 @@ protected bool IsShapeHit(WindowPixel[,] gameScreen, int x, int y)
                 y > gameScreen.GetUpperBound(1) ||
                 x < 0 ||
                 y < 0 ||
-                (gameScreen[x,y] != default && gameScreen[x, y].IsShape);
+                (gameScreen[x, y] != default && gameScreen[x, y].IsShape);
+        }
+
+        public override bool CanOverlap(BaseShape other)
+        {
+            return false; // Not Implemented Yet
+        }
+
+        public override void OnCollision(BaseShape other)
+        {
+            // Not Implemented Yet
         }
```

Comment:
After you merge with changes in dev, `MovingShape` will become abstract, so you can remove these impls

Reply:
Fixed